src/azure-cli/azure/cli/command_modules/profile/__init__.py

```diff
@@ -75,6 +75,7 @@ def load_arguments(self, command):
 
         with self.argument_context('account get-access-token') as c:
             c.argument('resource_type', get_enum_type(cloud_resource_types), options_list=['--resource-type'], arg_group='', help='Type of well-known resource.')
+            c.argument('tenant', options_list=['--tenant', '-t'], is_preview=True, help='Tenant ID for which the token is acquired. Only available for user and service principal account, not for MSI or Cloud Shell account')
```

Comment:
why do not we support MSI?

Reply:
Please see the PR description.

Reply:
The explanation is in the PR description.

Reply:
do we need to provide an example for this parameter ?

Reply:
@arrownj, good point. Added as requested.